### ISSUE TITLE
IOS Build Issue :' Razorpay/Razorpay-Swift.h' file not found

### ISSUE THREAD

**Issue by Nphadtare2021** [razorpay/react-native-razorpay#475]
## Description
Please provide a clear and concise description of what the bug is. Include screenshots if needed.
Please test using the latest React Native Razorpay plugin release to make sure your issue has not already been fixed. 

* **Specific to iOS Users :-**
- [Y ] I have tried updating Razorpay pod to the latest version by using 'pod update'.

## Razorpay Package Version :
"react-native-razorpay": "^2.3.0",

## Xcode Version (iOS) :
XCODE Version 13.4 (13F17a)

## Razorpay-pod version (iOS) :
pod 'razorpay-pod', '1.3.4'

## System Setup :
Versions:
react = 17.0.2
react-native = 0.64.4
react-native-razorpay = ^2.3.0,
CocoaPods = 1.11.3
razorpay-pod (1.3.4)
Xcode = Version 13.4 (13F17a)

### What you did:
i have integrated the razorpay  using COCOAPOD by adding " pod 'razorpay-pod', '1.3.4'" in pod file .
My build is getting successfully build using XCODE, but while I am trying to ARCHIVE i m getting ERROR!!

### What happened:

Got Following Error:
Lexical or Preprocessor Issue : 'Razorpay/Razorpay-Swift.h' file not found.
![Screenshot 2024-04-01 at 7 14 29 PM](https://github.com/razorpay/react-native-razorpay/assets/86679744/c90486e6-2834-4cb3-b73e-fdfedbcdd77c)

**Comment by vivekshindhe** [comment]
Hey @Nphadtare2021, I tried reproducing this but wasn't successful. It seems like a pod cache issue. We have set the react-native-razorpay project to pick the latest pod version directly. Could you try uninstalling the react-native-razorpay package and then reinstalling the package and running `pod install`? This should most likely resolve this issue for you.

**Comment by Nphadtare2021** [comment]
Hi @vivekshindhe , I had cleared the pod cache,cocoapods cache and system  XCODE derived data , still error not resolved

**Comment by vivekshindhe** [comment]
I tested this just before I responded above. Could you let me know if you tried to reinstall the package? @Nphadtare2021

**Comment by Nphadtare2021** [comment]
@vivekshindhe , Yes i have uninstall  package, then reinstall the  package followed by pod install.

**Comment by vivekshindhe** [comment]
@Nphadtare2021 can you share a sample project where you are able to reproduce the issue? Because we aren't able to reproduce the issue.

**Comment by malikzype** [comment]
Hi,

I have created a sample app to reproduce the same.

[my-project.zip](https://github.com/user-attachments/files/15766535/my-project.zip)

**Comment by vivekshindhe** [comment]
@malikzype looks like a regression issue in the new xcode version. The same can be found for multiple other libraries. One workaround I found for this for now is to set `Build Active Architecture Only` to `No` for both debug and prod builds. I was able to archive the project with that setting. Can you please try this once?  
Attaching the screenshot for your reference
![Screenshot 2024-06-11 at 5 03 45 PM](https://github.com/razorpay/react-native-razorpay/assets/57621906/4d213a4b-aadc-4438-a6fa-da1c921a210e)

**Comment by vivekshindhe** [comment]
@malikzype Hey, were you able to test this?
